src/tests/suite/daos_iotest.h

```diff
@@ -160,7 +167,8 @@ enum test_op_type {
 	TEST_OP_ADD		= 5,
 	TEST_OP_EXCLUDE		= 6,
 	TEST_OP_POOL_QUERY	= 7,
-	TEST_OP_MAX		= 7,
+	TEST_OP_SNAPSHOT_CREATE = 8,
```

Comment:
ah, you intend to create snapshot independently?  You probably need delete snapshot after verification or sometime later.